project_backend/Services/ItemService.cs

```diff
@@ -19,5 +19,29 @@ public List<Item> GetItems()
             }
             return itemsList;
         }
+        public async Task<bool> AddNewItem(int id, string name, decimal price, int quantity, string? created_by)
+        {
+            return await _itemRepository.AddNewItem(id, name, price, quantity, created_by) > 0;
+        }
+        public async Task<bool> UpdateItem(int id, string name, decimal price, int quantity)
```

Comment:
Reiktu pries vykdant koda patikrinti ar itemas su tokiu id egzistuoja, same su delete funkcija ir reiktu ismesti exceptionus (in case something we didn't think of goes wrong) :D

Reply:
noted:) sexceprionus atskiram branch'e darau, tai su kitu PR bus